src/components/InlineCheckbox/InlineCheckbox.js

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const InlineCheckbox = ({ id, checked, name, onClick, onKeyDown }) => (
+  <React.Fragment>
```

Comment:
Are we officially dropping React 15 support?

Reply:
I think when using DataTable, yeah. There is another Fragment in the DataTable story as well.

Reply:
OK let’s make sure we document that!

Reply:
Good call 👍